### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 - **v5.29.1, released 2020-12-03**
   - Disabled the xmlsec1-based SAML library in favor of the re-enabled and improved SAML library.
-  - Added ``UnreadMentions`` column to ``ThreadMemberships`` table. Also included additional bug fixes related to Collapsed Reply Threads feature, such as a fix for an issue where errors related to ``ThreadMemberships`` appeared in the server logs when replying to a user on a thread.
+  - Added ``UnreadMentions`` column to ``ThreadMemberships`` table to fix an issue where. . . . Also included additional bug fixes related to Collapsed Reply Threads feature, such as a fix for an issue where errors related to ``ThreadMemberships`` appeared in the server logs when replying to a user on a thread.
```
Comment (amyblais) [inline source/administration/changelog.md:11]: @esethna Are there any additional details we can add for the collapsed reply thread fixes? What specifically the issue fixes (i.e. what the impact is on users or admins), then followed by the technical details.

Comment (esethna) [reply source/administration/changelog.md:11]:
```
Added ``UnreadMentions`` column to ``ThreadMemberships`` table, and fixed server log warnings related to ``ThreadMemberships``. Admins planning to enable [Collapsed Reply Threads ](https://mattermost.com/blog/dev-sneak-peek-collapsed-reply-threads/) (available in beta in Q1 2021) are recommended to upgrade to v5.29.1 or later.
```

@amyblais, note that link will go live on Monday. If we need something in the interim, we can use this link: https://docs.google.com/presentation/d/1QSrPws3N8AMSjVyOKp15FKT7O0fGMSx8YidjSDS4Wng/edit#slide=id.g2f0aecc189_0_245

Comment (amyblais) [reply source/administration/changelog.md]: Thanks, updated.

**Review by esethna — APPROVED** [review]

**Comment by esethna** [comment]
@amyblais can we add a task to update the link to the blog post on Monday?

**Pull request merged**